service/awsconfig/v5/resource/loadbalancer/mocks.go

```diff
@@ -0,0 +1,50 @@
+package loadbalancer
```

Comment:
Just for sanity. If this is for testing only, please let's add `_test.go` suffix to the file name.

Reply:
+1 I think we should rename this. Do I miss something?

Reply:
Good point. The file is just for tests so I've renamed.